libs/core/langchain_core/tools.py

```diff
@@ -290,7 +290,7 @@ def create_schema_from_function(
     # Pydantic adds placeholder virtual fields we need to strip
     valid_properties = _get_filtered_args(inferred_model, func, filter_args=filter_args)
     return _create_subset_model(
-        f"{model_name}Schema",
```

Comment:
You need to take into account the users that might rely on this name.

Reply:
@hinthornw we could try and make this change as part of 0.3.

@RafaelXokito are you relying on this feature?

Reply:
@eyurtsev no I don't, thank you for asking.
I thought it would be good idea to  consider that scenario.